src/crud/routes/user.routes.ts

```diff
@@ -0,0 +1,89 @@
+import express, { Request, Response } from 'express'
+import * as UserService from '../services/user.service'
+import * as Validation from '../middleware/validation.middleware'
+import Joi from 'joi'
+import { User } from '../types'
+
+export const router = express.Router()
+
+const NO_USER_MSG = 'No such user'
+const DEFAILT_LIMIT = 5
+
+const userSchema: Joi.ObjectSchema<User> = Joi.object().keys({
+  login: Joi.string().alphanum().required(),
+  password: Joi.string().pattern(/^[a-zA-Z0-9]{3,30}$/).required(),
+  age: Joi.number().integer().min(4).max(130).required()
+})
+
+const getErrorMessage = (error: unknown): { message: string } => {
+  const message = error instanceof Error ? error.message : String(error)
+  return { message }
+}
+
+router.post('/createUser', Validation.validateSchema(userSchema), (req: Request, res: Response) => {
+  try {
```

Comment:
Let's create one more layer controllers. It will be better structure. Please take a look to this article for example  https://www.coreycleary.me/project-structure-for-an-express-rest-api-when-there-is-no-standard-way about structure of project.